File: Projects/grader/grader.py
class Dog:
    def __init__(self):  # this is a constructor
        name = input("What is your name?")
        age = input("What is your age?")
        grade = input("What is your grade?")
        letterGrade = self.myFunction(grade)
        if letterGrade != "F":
            print(f'Congrats {name}, you passed with a {letterGrade}!')
        else:
            print(f'Sorry, {name}. You seem to have failed.')

    def myFunction(grade):
        if grade < 60:
            return "F"
        elif grade >= 60 and grade < 70:
            return "D"
        elif grade >= 70 and grade < 80:
            return "C"
        elif grade >= 80 and grade < 90:
            return "B"
        else:
            return "A"


myDog = Dog()
# print(myDog.getAge())

File: Projects/grader/grader.rb
class Dog
    def initialize()
      name = getName()
      age = getAge()
      grade = getGrade()
      letterGrade = calcGrade(grade)
      if letterGrade != "F"
        puts "Congrats, #{name} you passed with a #{letterGrade}!"          
      else
        puts "Sorry, #{name}, you failed."
      end
  end
  def getName
    puts "What is your name?"
      @name=gets
      return @name
  end
  def getAge
    puts "What is your age?"
      @age=Integer(gets)
      return @age
  end
  def getGrade
    puts "What is your grade?"
      @grade=Integer(gets)
      return @grade
  end
  def calcGrade(grade)
    if grade < 60
        return "F"
    elsif grade >= 60 && grade < 70
        return "D"
    elsif grade >= 70 && grade < 80
        return "C"
    elsif grade >= 80 && grade < 90
        return "B"
    elsif grade >= 90
        return "A"
    end
  end
end
myDog = Dog.new